docs/tutorials/forum-application/6.main.md

```diff
@@ -0,0 +1,200 @@
+# Main function
+
+**This main function is responsible for running the Forum Application blockchain.**
+
+```go
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	app "github.com/andynog/abci2-forum-app/abci"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"syscall"
+
+	"github.com/cometbft/cometbft/p2p"
+	"github.com/cometbft/cometbft/proxy"
+	"github.com/spf13/viper"
+
+	db "github.com/cometbft/cometbft-db"
+	cfg "github.com/cometbft/cometbft/config"
+	cmtflags "github.com/cometbft/cometbft/libs/cli/flags"
+	cmtlog "github.com/cometbft/cometbft/libs/log"
+	nm "github.com/cometbft/cometbft/node"
+	"github.com/cometbft/cometbft/privval"
```

Comment:
Do we need to report all this code here in the tutorial itself? Can we rather present `main` with code snippets, as we've done above?